packages/components/src/molecules/NavbarSlider/NavbarSlider.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+import { SlideOver } from '../..'
+import type { SlideOverProps } from '../..'
+
+type RequiredSliderOverProps = 'isOpen' | 'direction' | 'size'
+
+export interface NavbarSliderProps
+  extends Omit<SlideOverProps, RequiredSliderOverProps> {
+  /**
+   * A boolean value that represents the state of the slider.
+   */
+  isOpen?: SlideOverProps['isOpen']
+  /**
+   * Represents the side that the slider comes from.
+   */
+  direction?: SlideOverProps['direction']
+  /**
+   * Represents the size of the slider.
+   */
+  size?: SlideOverProps['size']
+}
+
+function NavbarSlider({
+  children,
+  size = 'full',
+  isOpen = true,
+  direction = 'leftSide',
+  testId = 'fs-navbar-slider',
```

Comment:
```suggestion
 testId = 'fs-navbar-slider',
  children,
  size = 'full',
  isOpen = true,
  direction = 'leftSide',
```

Reply:
The Typescript standards recommend this order: props **without** default values -> props **with** default values -> prop spreading.
Is there any order for the props with default values you would recommend/follow?